Bonsai.Tinkerforge/BrickletDeviceNameConverter.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using Bonsai.Expressions;
+using Tinkerforge;
+using System.Threading;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Bonsai.Tinkerforge
+{
+    internal class BrickletDeviceNameConverter : TypeConverter
+    {
+        public Dictionary<string, TinkerforgeHelpers.DeviceData> devices;
+        //private Stopwatch stopwatch;
+
+        public override bool GetStandardValuesSupported(ITypeDescriptorContext context)
+        {
+            return true;
+        }
+
+        public override bool GetStandardValuesExclusive(ITypeDescriptorContext context)
+        {
+            return true;
```

Comment:
We did use to have this in drop-down converters, however we found out setting this override to `true` prevents users from manually typing values in the property, which can be annoying in situations where enumeration fails (i.e. `Exclusive` here means that the only allowed values in the property are the values listed in the drop down).

Probably best to remove this to keep the property flexible.